src/content/docs/distribute/flatpak.mdx

```diff
@@ -1,45 +1,99 @@
 ---
-title: Flatpak / Flathub
-draft: true
+title: Flathub
 sidebar:
   order: 1
-  badge:
-    text: WIP
-    variant: caution
 ---
 
+import { Tabs, TabItem, Card } from '@astrojs/starlight/components'
+
 For detailed information on how Flatpak works, you can read [Building your first Flatpak](https://docs.flatpak.org/en/latest/first-build.html)
 
 This guide assumes you want to distribute your Flatpak via [Flathub](https://flathub.org/), the most commonly used platform for Flatpak distribution. If you plan on using other platforms, please consult their documentation instead.
-
-TODO: deb vs from source
-
-## Build locally
+### Prerequisites
 
 To test your app inside the Flatpak runtime you can build the Flatpak locally first before uploading your app to Flathub. This can also be helpful if you want to quickly share development builds.
 
 **1. Install `flatpak` and `flatpak-builder`**
 
 To build Flatpaks locally you need the `flatpak` and `flatpak-builder` tools. For example on Ubuntu you can run this command:
+<Tabs>
+  <TabItem label="Debian">
 
 ```sh
+sudo apt update
```

Comment:
can we remove the update commands again? I'd prefer to keep the install command clean/small. (Thinking about doing the same for the initial tauri reqs docs but note sure yet)